src/commands/stac-validate/stac.validate.ts

```diff
@@ -69,14 +68,6 @@ export const commandStacValidate = command({
     const ajv = new Ajv.default({
       allErrors: true,
       strict: args.strict,
-      loadSchema: (uri: string): Promise<SchemaObject> => {
```

Comment:
Why was this considered dead code?

Reply:
I couldn't find any reference to it. There's a separate function by the same name which is called, but this one never seems to be.

Reply:
https://github.com/stac-utils/stac-node-validator/blob/master/index.js#L20C2-L20C12 implies it existed at one point?

Reply:
That does seem to reuse the function in the constructor. I still don't see anywhere it's called, although it could be within AJV itself. I'd expect our tests to suss that out, but maybe that code is untested.

Reply:
[This code](https://github.com/linz/argo-tasks/blob/0be61224024c2f4fc7263d9894766b88db9298e7/node_modules/ajv/dist/core.js#L162-L164) might indicate that it's being run.

Reply:
OIC, `src/commands/stac-validate/__test__/stac.validate.test.ts` doesn't test the validation functionality at all.

Reply:
you can never assume that something doesn't run just because tests don't cover it... especially when its inside of a third party library!

Reply:
Of course! I meant that I expected the tests to cover it, so it would be detected if it was called. But I'll just revert this rather than write another test.